webapps/pom.xml

```diff
@@ -150,42 +150,14 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>3.5.15</version>
+      <version>${version.mockito.new}</version>
```

Comment:
Why do we have `version.mockito` and `version.mockito.new`? Should we document the usage of both properties in the parent pom (where they are defined)?

Reply:
It's a temporary phase until all the modules are ready for the new version. After this and another PR I will remove the previous version and rename the `version.mockito.new` to `version.mockito`.

Reply:
Makes sense, thanks for explaining :)